benefits/oauth/hooks.py

```diff
@@ -35,6 +39,39 @@ def post_logout(cls, request):
         origin = session.origin(request)
         return redirect(origin)
 
+    @classmethod
+    @method_decorator(
+        [
+            decorator_from_middleware(AgencySessionRequired),
+            decorator_from_middleware(FlowSessionRequired),
+            decorator_from_middleware(LoginRequired),
+        ]
+    )
+    def claims_verified_eligible(cls, request, claims_request, claims_result):
+        super().claims_verified_eligible(request, claims_request, claims_result)
+        analytics.finished_sign_in(request)
+
+        flow = session.flow(request)
+        eligibility_analytics.started_eligibility(request, flow)
+
+        return verified(request)
+
+    @classmethod
+    @method_decorator(
+        [
+            decorator_from_middleware(AgencySessionRequired),
+            decorator_from_middleware(FlowSessionRequired),
+        ]
+    )
+    def claims_verified_not_eligible(cls, request, claims_request, claims_result):
+        super().claims_verified_not_eligible(request, claims_request, claims_result)
+        analytics.finished_sign_in(request, error=claims_result.errors)
+
+        flow = session.flow(request)
+        eligibility_analytics.started_eligibility(request, flow)
+
+        return redirect(reverse(routes.ELIGIBILITY_UNVERIFIED))
```

Comment:
We don't need the call to `reverse` here; [`redirect`](https://docs.djangoproject.com/en/5.1/topics/http/shortcuts/#redirect) can handle an `app:endpoint` pattern and will call it for us.